### PR TITLE
test(annotation): cover distribute_whole_files

### PR DESCRIPTION
**Pull request by UkiStefanovic**
The function distribute_whole_files from module annotation/distribution/main.py already had tests, but did not have full coverage. This pull request covers the whole function with 3 more tests that are edgecases. You can find these tests in annotation/tests/test_distribution.py

### CODE REVIEW

**Review by Negoje — COMMENTED** [review]
Looks fine to me

File: annotation/tests/test_distribution.py [hunk 1]
```diff
@@ -552,6 +552,69 @@ def test_distribute_annotation_whole_files(
...
+            [{"user_id": 0, "pages_number": 0}],
+            0,
+            False,
+            TaskStatusEnumSchema.pending,
+        )
+        == []
+    )
+
+
+def test_distribute_whole_files_user_page_correction_edge_case():
```
Comment (cakeinsauce) [inline annotation/tests/test_distribution.py:569]: remove edge_case

File: annotation/tests/test_distribution.py [hunk 2]
```diff
@@ -552,6 +552,69 @@ def test_distribute_annotation_whole_files(
...
+                "is_validation": False,
+                "job_id": 0,
+                "pages": [1, 2, 3, 4, 5],
+                "status": TaskStatusEnumSchema.pending,
+                "user_id": 0,
+            },
+        ]
+
+
+def test_distribute_whole_files_break_edge_case():
```
Comment (cakeinsauce) [inline annotation/tests/test_distribution.py:597]: remove edge_case

**Review by UkiStefanovic — COMMENTED** [review]
I fixed the code

**Review by cakeinsauce — APPROVED** [review]

**Pull request merged**